apps/nextcloud/templates/services/app/_env.yml.j2

```diff
@@ -0,0 +1,16 @@
+POSTGRES_DB: "{{ nextcloud_database_name }}"
+POSTGRES_HOST: "nextcloud-{{ nextcloud_database_host }}-{{ deployment_stamp }}"
+POSTGRES_PORT: "{{ nextcloud_database_port }}"
+NEXTCLOUD_ADMIN_USER: "{{ nextcloud_admin_user }}"
+NEXTCLOUD_CURRENT_DOMAIN: "{{ nextcloud_host }}"
+NEXTCLOUD_PREVIOUS_DOMAIN: "previous.{{ nextcloud_host }}"
+NEXTCLOUD_NEXT_DOMAIN: "next.{{ nextcloud_host }}"
+NEXTCLOUD_LOG_LEVEL: "{{ nextcloud_log_level }}"
+NEXTCLOUD_LOG_TYPE: "{{ nextcloud_log_type }}"
+NEXTCLOUD_REDIS_HOST: "{{ nextcloud_redis_host }}"
+NEXTCLOUD_REDIS_PORT: "{{ nextcloud_redis_port }}"
+NEXTCLOUD_DEBUG: "{{ nextcloud_debug }}"
+NEXTCLOUD_VERSION: "{{ nextcloud_version }}"
+USER_NAME: "{{ nextcloud_container_user }}"
+HOME: "{{ nextcloud_container_home_dir }}"
+NEXTCLOUD_BASE_DIR: "{{ nextcloud_base_dir }}"
```

Comment:
Should we sort all of them alphabetically?

Reply:
Do you have a command to do that ?

Reply:
I group them by suffix, "type" and then alphabetically

Reply:
`sort` unix command is what your are looking for. Or else `vim` can do that :trollface:

Reply:
Thanks but I prefer like I sorted